content/intro-to-storybook/angular/en/using-addons.md

```diff
@@ -56,9 +56,9 @@ Problem solved! The text is now truncated when it reaches the boundary of the Ta
 
 ### Adding a new story to avoid regressions
 
-In the future, we can manually reproduce this problem by entering the same string via Controls. But it's easier to write a story that showcases this edge case. That expands our regression test coverage and clearly outlines the limits of the component(s) for the rest of the team.
+In the future, we can manually reproduce this problem by entering the same string via Controls. But it's easier to write a story that showcases this edge case. That expands our regression test coverage and clearly outlines the component(s) limits for the rest of the team.
```

Comment:
I think this was more clear before.

Reply:
Thanks for the clarification. Fixed with the latest commits